blog/news-and-updates/team-spotlight-execution.mdx

```diff
@@ -0,0 +1,86 @@
+ 
+*The team consists of 5 engineers: Alexandra, Dimitris, Dragoljub, Islam, Maksym and 1 researcher: Bogdan. All team members are based in Switzerland.*
+
+**Let's dive a bit into how execution on the IC works. On a fundamental level, what is the IC’s execution environment? What happens in it?**
+
+*The execution environment is responsible for processing messages (either from users or canisters), keeping track of cycles for canisters and updating their state after message execution completes.*            
+
+**Could you explain how the execution layer fits into the IC stack?**
+
+*The execution layer is the top layer of the core IC stack. It’s part of what we call the “replicated state machine” which additionally includes message routing. Messages are delivered in batches through consensus to message routing and the execution environment is responsible for processing them.*
+
+
+**What is the process of an execution round on the IC?**
+
+*An execution round proceeds in several phases.*
+
+*During the preparation phase, some cleanup/bookkeeping tasks are performed, including figuring out which canisters are long-running (i.e. require DTS execution) and purging expired ingress messages from canister queues.*
+
+*In the next phase messages to the management canister are processed. Following that the main execution phase begins where the scheduling algorithm will assign canisters to different cores to execute and will start processing as many messages as possible within the round.*
+
+*The finalization phase includes some cleanup/bookkeeping tasks, including charging canisters for resources, pruning messages from ingress history, processing stopping canisters, updating some metrics and checking various invariants about the state of the system.*
+
+
+**Thank you for that description! I think that is very insightful for developers to understand what's happening behind the scenes. How do developers interact with the execution layer through canisters?**
+
+*Developers interact with the execution environment through two sets of APIs: the first is a synchronous one which we call the system API. This is analogous to system calls on a regular operating system. The second class is the API provided by the management canister which is asynchronous and is very similar to interacting with any other canister on the IC – this API allows canisters to manage other canisters as well as get access to some of the special features of the IC, including tECDSA and HTTP outcalls.*
+
+**How does the Execution team work with other teams?**
+
+*The Execution team works with several other teams within DFINITY. We work pretty closely with the Runtime team who are in charge of the lower level Wasm execution engine of the IC, the Message Routing team who are responsible for routing of messages to the appropriate subnet for execution and state management as well as the SDK and languages teams to expose the APIs provided by the execution environment to the developers.*
```

Comment:
should Languages team be capitalized here?

Reply:
Yes, thanks for catching that!